src/niscope/templates/session.py/fancy_fetch.py.mako

```diff
@@ -19,6 +19,11 @@
             if num_samples is None:
                 num_samples = self.horz_record_length
 
+        channel_names = _converters.expand_channel_string(
```

Comment:
you made the same fix four times, there's gotta be a better way of factoring out duplicate code, no?